resources/testharness.js

```diff
@@ -3379,13 +3377,10 @@ policies and contribution forms [3].
     function assert(expected_true, function_name, description, error, substitutions)
     {
         if (tests.tests.length === 0) {
```

Comment:
Can’t we delete this code and let there be a harness error in case the assert fails? To catch stray asserts only before the first test is defined does not seem that useful to me, compared to the cost of keeping this code.

Reply:
Sure